index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Portfolio page">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+        integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="styles.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <ul class="navbar">
+                <li class="logo">
+                    <a href="#">Welcome</a>
```

Comment:
- [ ] [OPTIONAL] Great job with the design. I think there are some small issues remaining. It would be better if you update the style of this link to match the Figma template. ✍️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/219363709-d82c2fdf-6b6c-4353-ae94-c824beb89d0b.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/219363711-0bc5b027-7206-4bcc-aff3-00f3b6b0744c.png" width="300">|

Reply:
Okay. On it.